docker/build.sh

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CSTOR_ORG="mayadata-io"
```

Comment:
Can we have this also passed via the github action. same like did in other files. Or set the value to `mayadata-io` only if the CSTOR_ORG variable is unset.

Reply:
making it fail will be more assertive. made the changes.